cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -76,7 +77,42 @@ func main(clusterID string) error {
 	}
 
 	fmt.Println(query.Build())
-	fmt.Println("\nLink to Web Console - \n", GetLinkToWebConsole(hcpCluster.DynatraceURL, since, base64.StdEncoding.EncodeToString([]byte(query.finalQuery))))
+	SearchQuery := map[string]interface{}{
+		"version": 0,
+		"data": map[string]interface{}{
+			"tableConfig": map[string]interface{}{
+				"visibleColumns": []string{"timestamp", "status", "content"},
+				"columnAttributes": map[string]interface{}{
+					"columnWidths": map[string]interface{}{},
+					"lineWraps": map[string]interface{}{
+						"timestamp": true,
+						"status":    true,
+						"content":   true,
+					},
+					"tableLineWrap": true,
+				},
+				"columnOrder": []string{"timestamp", "status", "content"},
+			},
+			"queryConfig": map[string]interface{}{
+				"query":     query.finalQuery,
+				"timeframe": map[string]interface{}{"from": fmt.Sprintf("now()-%vh", since), "to": "now()"},
+				"filter": map[string]interface{}{
+					"datatype": "logs",
+					"filters":  map[string]interface{}{},
+					"sort": map[string]interface{}{
+						"field":     "timestamp",
+						"direction": "desc",
+					},
+				},
+				"showDqlEditor": true,
+			},
+		},
+	}
+	mStr, err := json.Marshal(SearchQuery)
+	if err != nil {
+		return fmt.Errorf("failed to Marshal json %v", err)
```

Comment:
The error message needs to improve to atleast let know user where it actually failed. Maybe something like `failed to create JSON for sharable URL`